external/glfw/js/library_glfw.js

```diff
@@ -566,20 +566,19 @@ var LibraryGLFW = {
       }
     },
 
-    //adaptation for our needs of https://github.com/emscripten-core/emscripten/blob/941bbc6b9b35d3124f17d2503d7a32cc81032dac/src/library_glfw.js#L662
```

Comment:
I removed this comment with a reference to the old commit - is it OK? To make the code cleaner. It's known that GLFW in Defold is a patchwork of bits and pieces from everywhere, so there's no point in this comment.